### ISSUE TITLE
`Set target of entity` error

### ISSUE THREAD

**Issue by Rigantion** [SkriptLang/Skript#5950]
### Skript/Server Version

```markdown
Skript-2.7.0-beta3
```

### Bug Description

This code worked before but when i tried to update my skript version, my skript got error

```
spawn 1 of iron golem at {SV::Location::%arg-2%}
set last spawned entity's target to {_p}
force last spawned entity attack {_p}
```

Error Message
```
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! [Skript] Severe Error:
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! Something went horribly wrong with Skript.
[12:07:17 ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[12:07:17 ERROR]: #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
[12:07:17 ERROR]: #!#! Here is full list of them:
[12:07:17 ERROR]: #!#! SkBee v2.17.0 (https://github.com/ShaneBeee/SkBee) SkQuery v4.1.10
[12:07:17 ERROR]: #!#! We could not identify which of those are specially related, so this might also be Skript issue.
[12:07:17 ERROR]: #!#! You should try disabling those plugins one by one, trying to find which one causes it.
[12:07:17 ERROR]: #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
[12:07:17 ERROR]: #!#! In that case, you will be given instruction on how should you report it.
[12:07:17 ERROR]: #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
[12:07:17 ERROR]: #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! Stack trace:
[12:07:17 ERROR]: #!#! java.lang.UnsupportedOperationException
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.lang.util.SimpleExpression.change(SimpleExpression.java:238)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.expressions.ExprTarget.change(ExprTarget.java:130)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.effects.EffChange.execute(EffChange.java:280)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.lang.Effect.run(Effect.java:50)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:62)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:90)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.lang.Trigger.execute(Trigger.java:52)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.command.ScriptCommand.execute2(ScriptCommand.java:312)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.command.ScriptCommand.lambda$execute$0(ScriptCommand.java:277)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.command.ScriptCommand.execute(ScriptCommand.java:282)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.command.Commands.handleCommand(Commands.java:193)
[12:07:17 ERROR]: #!#!     at Skript-2.7.0-beta3.jar//ch.njol.skript.command.Commands$1.onPlayerCommand(Commands.java:152)
[12:07:17 ERROR]: #!#!     at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:40)
[12:07:17 ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:81)
[12:07:17 ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[12:07:17 ERROR]: #!#!     at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54)
[12:07:17 ERROR]: #!#!     at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126)
[12:07:17 ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2274)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.network.PlayerConnection.lambda$handleChatCommand$20(PlayerConnection.java:2258)
[12:07:17 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.b(IAsyncTaskHandler.java:59)
[12:07:17 ERROR]: #!#!     at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.TickTask.run(TickTask.java:18)
[12:07:17 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[12:07:17 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:24)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1339)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197)
[12:07:17 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.x(IAsyncTaskHandler.java:126)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.bg(MinecraftServer.java:1316)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1309)
[12:07:17 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.c(IAsyncTaskHandler.java:136)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.p_(MinecraftServer.java:1287)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1175)
[12:07:17 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318)
[12:07:17 ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! Version Information:
[12:07:17 ERROR]: #!#!   Skript: 2.7.0-beta3 (latest)
[12:07:17 ERROR]: #!#!     Flavor: skriptlang-github
[12:07:17 ERROR]: #!#!     Date: 03:18:03.366646700
[12:07:17 ERROR]: #!#!   Bukkit: 1.20.1-R0.1-SNAPSHOT
[12:07:17 ERROR]: #!#!   Minecraft: 1.20.1
[12:07:17 ERROR]: #!#!   Java: 17.0.8 (Java HotSpot(TM) 64-Bit Server VM 17.0.8+9-LTS-211)
[12:07:17 ERROR]: #!#!   OS: Windows 10 amd64 10.0
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! Server platform: Paper
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! Current node: null
[12:07:17 ERROR]: #!#! Current item: set target of (the last spawned entity >> ch.njol.skript.classes.data.DefaultConverters$$Lambda$6685/0x000001afacf8d420@2f9e84f5: ConverterInfo{from=interface org.bukkit.entity.Entity,to=interface org.bukkit.entity.LivingEntity,converter=ch.njol.skript.classes.data.DefaultConverters$$Lambda$6685/0x000001afacf8d420@2f9e84f5,flag=0}) to {_p} (as org.bukkit.entity.LivingEntity)
[12:07:17 ERROR]: #!#! Current trigger: command /sv2 (simple event) (SV2.sk, line 32)
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! Thread: Server thread
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! Language: english
[12:07:17 ERROR]: #!#! Link parse mode: DISABLED
[12:07:17 ERROR]: #!#!
[12:07:17 ERROR]: #!#! End of Error.
[12:07:17 ERROR]: #!#!
```

### Expected Behavior

The iron golem targetting player

### Steps to Reproduce

I updated my skript from Skript 2.16.5 to 2.17.0 beta-3

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by Fusezion** [comment]
A fix for this has been made and merged #5856. Will be fixed in the actual release of 2.7.
Duplicate of #5761

**Comment by AyhamAl-Ali** [comment]
Duplicate of #5761
